modules/cms-kit/src/Volo.CmsKit.Domain.Shared/Volo/CmsKit/Settings/AppSettings.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace Volo.CmsKit.Settings;
+
+public static class AppSettings
```

Comment:
AppSettings is not a good name. First of all, this is a module, not application. You can use `CmsKitSettings` for example.